internal/cephfs/mounter/fuse.go

```diff
@@ -65,10 +68,25 @@ func mountFuse(ctx context.Context, mountPoint string, cr *util.Credentials, vol
 	if volOptions.FsName != "" {
 		args = append(args, "--client_mds_namespace="+volOptions.FsName)
 	}
-
-	_, stderr, err := util.ExecCommand(ctx, "ceph-fuse", args[:]...)
-	if err != nil {
-		return fmt.Errorf("%w stderr: %s", err, stderr)
+	var (
+		stderr string
+		err    error
+	)
+
+	netPath, nErr := util.GetNetNamespaceFilePath(util.CsiConfigFile, volOptions.ClusterID)
```

Comment:
would it not make sense to detect this in `NodeStageVolume` and set the `netPath` as part of the `VolumeOptions` struct? The `netPath` is related to how the connection is made, similar to the MONs and other options.

Reply:
Yes it should be possible, updated code for it.